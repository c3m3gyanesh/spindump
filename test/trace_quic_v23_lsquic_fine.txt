Three fine runs of 10K file transfer with Lsquic to a Picoquic server on the Internet. Using the command "http_client -H test.privateoctopus.com -s 34.238.92.221:4433 -p /10000". This worked well, given that the -H option was now supplied (needs to be supplied, otherwise the client would crash).
