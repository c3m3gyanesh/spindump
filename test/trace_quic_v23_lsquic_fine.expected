QUIC 172.16.114.251 <-> 34.238.92.221 08083c2c84b54986-194008f8cbb18d4562f5ac73298497f0885c06 (62938:4433) at 1571159979007734 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 08083c2c84b54986-083f8389c0cf790c (62938:4433) at 1571159979091440 measurement up right 83706 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979093036 spinvalue up 0 responder packets 1 3 bytes 1228 3788 bandwidth 1228 3788 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979095521 spinvalue up 1 initiator packets 2 4 bytes 1560 5186 bandwidth 1560 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979106691 spinvalue up 1 initiator packets 3 4 bytes 1665 5186 bandwidth 1665 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979106692 spinvalue up 1 initiator packets 4 4 bytes 1726 5186 bandwidth 1726 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979119269 spinvalue up 1 initiator packets 5 4 bytes 1826 5186 bandwidth 1826 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979175736 spinvalue up 1 responder packets 6 5 bytes 1926 5319 bandwidth 1926 5319 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979175740 spinvalue up 1 responder packets 6 6 bytes 1926 6717 bandwidth 1926 6717 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979176319 spinvalue up 0 initiator packets 6 7 bytes 1926 8115 bandwidth 1926 8115 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979176319 measurement up left 583 packets 6 7 bytes 1926 8115 bandwidth 1926 8115 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979179619 spinvalue up 1 responder packets 7 7 bytes 1986 8115 bandwidth 1986 8115 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979179627 spinvalue up 1 responder packets 7 8 bytes 1986 9513 bandwidth 1986 9513 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979180057 spinvalue up 0 initiator packets 7 9 bytes 1986 10911 bandwidth 1986 10911 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979183631 spinvalue up 1 responder packets 8 9 bytes 2046 10911 bandwidth 2046 10911 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979185431 spinvalue up 1 responder packets 8 10 bytes 2046 12309 bandwidth 2046 12309 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979185436 spinvalue up 1 responder packets 8 11 bytes 2046 13707 bandwidth 2046 13707 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979185849 spinvalue up 0 initiator packets 8 12 bytes 2046 15105 bandwidth 2046 15105 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979189740 spinvalue up 1 responder packets 9 12 bytes 2106 15105 bandwidth 2106 15105 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979190037 spinvalue up 0 initiator packets 9 13 bytes 2106 15832 bandwidth 2106 15832 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979195598 spinvalue up 1 responder packets 10 13 bytes 2164 15832 bandwidth 2164 15832 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979266534 spinvalue up 0 responder packets 10 14 bytes 2164 15915 bandwidth 2164 15915 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979266534 measurement up right 90215 packets 10 14 bytes 2164 15915 bandwidth 2164 15915 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 083c2c84b5498614-083f8389c0cf790c (62938:4433) at 1571159979266534 measurement up full (responder) 90798 packets 10 14 bytes 2164 15915 bandwidth 2164 15915 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 08e0758b3c04c62a-219d12393f7ab5db71bac7859089f9a01329dd (49434:4433) at 1571159982496882 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 08e0758b3c04c62a-088621afe847a002 (49434:4433) at 1571159982577004 measurement up right 80122 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982577312 spinvalue up 0 responder packets 1 3 bytes 1228 3788 bandwidth 1228 3788 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982581409 spinvalue up 1 initiator packets 2 4 bytes 1304 5186 bandwidth 1304 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982582808 spinvalue up 1 initiator packets 4 4 bytes 1662 5186 bandwidth 1662 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982657466 spinvalue up 1 responder packets 7 5 bytes 2018 5266 bandwidth 2018 5266 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982657473 spinvalue up 1 responder packets 7 6 bytes 2018 6546 bandwidth 2018 6546 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982657527 spinvalue up 1 responder packets 7 7 bytes 2018 7826 bandwidth 2018 7826 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982658903 spinvalue up 0 initiator packets 7 8 bytes 2018 9106 bandwidth 2018 9106 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982658903 measurement up left 1437 packets 7 8 bytes 2018 9106 bandwidth 2018 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982658904 spinvalue up 0 initiator packets 8 8 bytes 2078 9106 bandwidth 2078 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982662643 spinvalue up 1 responder packets 9 8 bytes 2322 9106 bandwidth 2322 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982662830 spinvalue up 1 responder packets 9 9 bytes 2322 10386 bandwidth 2322 10386 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982663202 spinvalue up 0 initiator packets 9 10 bytes 2322 11666 bandwidth 2322 11666 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982667306 spinvalue up 1 responder packets 10 10 bytes 2382 11666 bandwidth 2382 11666 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982673031 spinvalue up 1 responder packets 10 11 bytes 2382 12946 bandwidth 2382 12946 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982673038 spinvalue up 1 responder packets 10 12 bytes 2382 14226 bandwidth 2382 14226 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982673712 spinvalue up 0 initiator packets 10 13 bytes 2382 15506 bandwidth 2382 15506 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982678043 spinvalue up 1 responder packets 11 13 bytes 2442 15506 bandwidth 2442 15506 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982678454 spinvalue up 0 initiator packets 11 14 bytes 2442 15841 bandwidth 2442 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982732445 spinvalue up 0 responder packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982732445 measurement up right 73542 packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982732445 measurement up full (responder) 74979 packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 e0758b3c04c62aa6-088621afe847a002 (49434:4433) at 1571159982753050 spinvalue up 0 responder packets 12 15 bytes 2500 15924 bandwidth 2500 15924 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 08a1c0418de6ac29-6666482d6cfa722d496f8efbfa1c9a907781fb (49158:4433) at 1571159986267888 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 08a1c0418de6ac29-08d6e182834cfaed (49158:4433) at 1571159986348489 measurement up right 80601 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986348824 spinvalue up 0 responder packets 1 3 bytes 1228 3788 bandwidth 1228 3788 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986352910 spinvalue up 1 initiator packets 2 4 bytes 1304 5186 bandwidth 1304 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986354310 spinvalue up 1 initiator packets 4 4 bytes 1662 5186 bandwidth 1662 5186 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986429425 spinvalue up 1 responder packets 7 5 bytes 2018 5266 bandwidth 2018 5266 note "V23,no spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986429432 spinvalue up 1 responder packets 7 6 bytes 2018 6546 bandwidth 2018 6546 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986429796 spinvalue up 1 responder packets 7 7 bytes 2018 7826 bandwidth 2018 7826 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986430258 spinvalue up 0 initiator packets 7 8 bytes 2018 9106 bandwidth 2018 9106 note "V23,no I-spin"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986430258 measurement up left 833 packets 7 8 bytes 2018 9106 bandwidth 2018 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986430259 spinvalue up 0 initiator packets 8 8 bytes 2078 9106 bandwidth 2078 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986434896 spinvalue up 1 responder packets 9 8 bytes 2322 9106 bandwidth 2322 9106 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986434903 spinvalue up 1 responder packets 9 9 bytes 2322 10386 bandwidth 2322 10386 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986435637 spinvalue up 0 initiator packets 9 10 bytes 2322 11666 bandwidth 2322 11666 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986438099 spinvalue up 1 responder packets 10 10 bytes 2382 11666 bandwidth 2382 11666 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986443824 spinvalue up 1 responder packets 10 11 bytes 2382 12946 bandwidth 2382 12946 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986443832 spinvalue up 1 responder packets 10 12 bytes 2382 14226 bandwidth 2382 14226 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986444267 spinvalue up 0 initiator packets 10 13 bytes 2382 15506 bandwidth 2382 15506 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986448783 spinvalue up 1 responder packets 11 13 bytes 2442 15506 bandwidth 2442 15506 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986449068 spinvalue up 0 initiator packets 11 14 bytes 2442 15841 bandwidth 2442 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986504683 spinvalue up 0 responder packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986504683 measurement up right 74425 packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986504683 measurement up full (responder) 75258 packets 12 14 bytes 2500 15841 bandwidth 2500 15841 note "V23,spinning"
QUIC 172.16.114.251 <-> 34.238.92.221 a1c0418de6ac2902-08d6e182834cfaed (49158:4433) at 1571159986523360 spinvalue up 0 responder packets 12 15 bytes 2500 15924 bandwidth 2500 15924 note "V23,spinning"
